screens/Menu/NearbyInterestDetails.js

```diff
@@ -1,33 +1,151 @@
 import React from "react";
-import { View, Text, StyleSheet } from "react-native";
+import { View, Text, StyleSheet, Image, SafeAreaView, AsyncStorage } from "react-native";
 import { Icon, Button } from "native-base";
+import StarRating from "react-native-star-rating";
 
 
+/**
+ * US34 - As a user, I would like to see the nearest outdoor points of interest #14
+ * US35 - As a user, I would like to get the direction to the chosen nearest point of interest #15
+ * US36 - As a user, I would like to see a detailed description of the selected places #29
+ * 
+ * Description: This screen will presents the layout 
+ * for the display of the point of interests. The points 
+ * of interest information will be retrieved from the Google Places API.
+ * The screen is composed on a flatList to create a grid, and have 
+ * an pressable item for each points of interest
+ */
+
 /**Prop passed
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 
 function NearbyInterestDetails(props) {
+/**
+ * navigation that serves to go back to previous screen
+ * @param  {} =>{props.navigation.goBack(
+ */
 
     const goBack = () => {
         props.navigation.goBack();
     }
 
-    const NearbyInterestDetailsScreen = true; 
+    const photoref = props.navigation.getParam("photoref", null);
```

Comment:
Why don't you just have an object that has all those attributes and you pass that instead? That's what I did for _PreviewDirections_ to _Directions_. In that case, you would call the getParam only once so you could deconstruct the object received here.

Reply:
Yes!

I do have an object that takes in all the props (from _NearbyInterest_). However the retrieval of the props, I'm not using the deconstructing way of _getParam_. 

(Been used to it since previous issues we had at first with the unit testing.)

I could change it.

Reply:
Oh, I was not talking about the other way to deconstruct getParam. I was talking about passing an object (as you said you have it) that contains _**all**_ those attributes. So you only do one getParam!

Reply:
Done!